src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,18 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ( {todos} ) => (
+  <>
+    <h1 className="App__title">Static list of todos</h1>
+    <section className="TodoList">
+      {todos.map(todo => (
+        <>
+          <TodoInfo
+            todo={todo}
+            key={todo.id}
+          />
+        </>
```

Comment:
there is  no need in React.Fragment 
```suggestion
          <TodoInfo
            todo={todo}
            key={todo.id}
          />
```